web/src/lightbox.js

```diff
@@ -190,6 +190,9 @@ export function render_lightbox_list_images(preview_source) {
 
             $image_list.append($node);
 
+            const $img_element = $("#lightbox_overlay .image-preview > .zoom-element > img");
+            $img_element.attr("src", img.src);
+
```

Comment:
LGTM Thanks!

Reply:
But why is this happening inside a loop?

Reply:
> But why is this happening inside a loop? I think this is the code for the list of all images in the feed at the bottom of the lightbox.

Corrected the code. Resetting the src in display_image() function works as well.